cmd/manager/main.go

```diff
@@ -215,6 +210,19 @@ func addMetrics(ctx context.Context, cfg *rest.Config, namespace string) {
 		log.Info("could not create metrics Service", "error", err.Error())
 	}
 
+	oclient, err := k8sclient.NewForConfig(cfg)
+	if err != nil {
+		log.Info("could not create config clientset", "error", err.Error())
```

Comment:
Please use `log.Error()`. I am not sure why the SDK generated code that used `log.Info()` for error cases.